ui/src/alerting/components/builder/AlertBuilder.scss

```diff
@@ -25,13 +25,13 @@
   }
 
   &.alert-builder--meta-card {
-    flex: 1 0 320px !important;
+    flex: 1 0 280px !important;
```

Comment:
why not make these percentages?

Reply:
% widths are so 2009 russ, learn flexbox

Reply:
🤣